src/index.ts

```diff
@@ -3,3 +3,9 @@ import './configure';
 
 export * from './types';
 export * from './components';
+
+export {createThemeContext} from './utilities/theme-provider';
+export {
```

Comment:
As I mentioned above I think we should move to a position where we selectively export our components since we have a lot of stuff being exported that probably shouldn't be